browser/ui/webui/settings/brave_default_extensions_handler.cc

```diff
@@ -372,6 +380,17 @@ void BraveDefaultExtensionsHandler::IsWidevineEnabled(
 #endif
 }
 
+void BraveDefaultExtensionsHandler::OnWalletTypeChanged() {
+  if (brave_wallet::GetDefaultWallet(profile_->GetPrefs())
+    != brave_wallet::mojom::DefaultWallet::BraveWallet)
```

Comment:
Shoudn't this be != brave_wallet::mojom::DefaultWallet::CryptoWallets?

Reply:
updated as discussed in slack, thanks!